noir-projects/noir-protocol-circuits/crates/types/src/address/aztec_address.nr

```diff
@@ -65,6 +65,15 @@ impl Deserialize<AZTEC_ADDRESS_LENGTH> for AztecAddress {
     }
 }
 
+impl Packable<AZTEC_ADDRESS_LENGTH> for AztecAddress {
+    fn pack(self) -> [Field; AZTEC_ADDRESS_LENGTH] {
+        [self.to_field()]
+    }
+
+    fn unpack(fields: [Field; AZTEC_ADDRESS_LENGTH]) -> Self {
+        AztecAddress::from_field(fields[0])
+    }
```

Comment:
Use ser

Reply:
Same as above. Packable required for public storage